VATSpy.dat

```diff
@@ -17618,7 +17618,7 @@ EGTT-C|London ACC (Central) - London|LON_E|EGTT-C
 EGTT-C|London ACC (Central) - London|LTC_N|EGTT-C
 EGTT-C|London ACC (Central) - London|LTC_NW|EGTT-C
 EGTT-N|London ACC (North) - London|LON_N|EGTT-N
-EGTT-N|London ACC (North) - London|MAN|EGTT-N
+EGTM|Manchester TMA (Up to FL285)|MAN|EGTM
 EGTT-N|London ACC (North) - London|MAN_E|EGTT-N

```

Comment:
Should this line not be removed too?

Reply:
Fixed in 708a0b1a41129bb55a6002fd1320df4258f35e96